modules/mod_ginger_rdf/models/m_rdf.erl

```diff
@@ -366,3 +366,7 @@ literal_object_value(#rdf_value{value = Value}) ->
     Value;
 literal_object_value(Other) ->
     Other.
+
+-spec unique([any()]) -> [any()].
+unique(Ids) ->
+    sets:to_list(sets:from_list(Ids)).
```

Comment:
sets:to_list etc doesn't preserve the original list order.

Reply:
Fixed in #517.